annotation/app/tasks/services.py

```diff
@@ -381,3 +389,109 @@ def unblock_validation_tasks(db: Session, task: ManualAnnotationTask) -> None:
             {"status": TaskStatusEnumSchema.ready}, synchronize_session=False
         )
     )
+
+
+def get_task_stats_by_id(
+    db: Session,
+    task_id: int,
+) -> Optional[AnnotationStatistics]:
+    return (
+        db.query(AnnotationStatistics)
+        .filter(AnnotationStatistics.task_id == task_id)
+        .first()
+    )
+
+
+def add_task_stats_record(
+    db: Session,
+    task_id: int,
+    stats: AnnotationStatisticsInputSchema,
+) -> AnnotationStatistics:
+    stats_db = get_task_stats_by_id(db, task_id)
+
+    if stats_db:
+        for name, value in stats.dict().items():
+            setattr(stats_db, name, value)
+        stats_db.updated = datetime.now()
```

Comment:
```suggestion
        stats_db.updated = datetime.utcnow()
```

Reply:
done as suggested, also required to change model defaults